course_schedule.py

```diff
@@ -0,0 +1,33 @@
+# Time Complexity: O(n) where n is the number of courses
```

Comment:
Check your complexity here. In graph you always express your complexity in terms of V and E